packages/dashboard/src/app/views/myStories/content/index.js

```diff
@@ -86,15 +87,8 @@ function Content({
               size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
               as="h3"
             >
-              {search?.keyword
-                ? sprintf(
-                    /* translators: %s: search term. */
-                    __(
-                      'Sorry, we couldn\'t find any results matching "%s"',
-                      'web-stories'
-                    ),
-                    search.keyword
-                  )
+              {search?.keyword || Boolean(Object.keys(filtersObject))
+                ? __("Sorry, we couldn't find any results.", 'web-stories')
```

Comment:
I think it is better we keep on the old message.

Reply:
Sounds good! I can implement that per the specific cases. Thanks!

Reply:
See: https://github.com/GoogleForCreators/web-stories-wp/pull/11738/commits/96e74e155faf9ff15d99f24a4862b381e680192c